src/implementations/vanilla/components/language-list/_language-list.scss

```diff
@@ -8,16 +8,16 @@
 @use '@ecl/vanilla-layout-grid/mixins/breakpoints';
 
 $_background-header: map.get(theme.$color, 'white-100');
-$_background-container: map.get(theme.$color, 'blue-100');
+$_background-container: null !default;
 $_background-link: map.get(theme.$color, 'white-100');
 $_background-link-hover: map.get(theme.$color, 'yellow-100');
 $_background-link-active: map.get(theme.$color, 'yellow-100');
 $_color-link-active: map.get(theme.$color, 'black-100');
-$_color-title: map.get(theme.$color, 'white-100');
+$_color-title: null !default;
 $_logo-height-desktop: 5.5rem;
-$_logo-width-desktop: 11.25rem;
+$_logo-width-desktop: null !default;
 $_logo-height-mobile: 3.75rem;
-$_logo-width-mobile: 7.5rem;
+$_logo-width-mobile: null !default;
 
```

Comment:
the color of the close button is not correct on EU
![image](https://user-images.githubusercontent.com/25579221/103898095-b7279900-50f4-11eb-8012-3fb7e2d82427.png)
I guess it will require an additional variable

Reply:
Thanks for spotting this, Updated using existing `$_color-title` variable.